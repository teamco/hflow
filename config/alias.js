const { NODE_ENV } = process.env;

const isDevelopment = NODE_ENV === 'development';

const alias = {
  worker: '/public/worker',
  assets: '/public/assets',
  components: '/src/components/',
  layouts: '/src/layouts/',
  locales: '/src/locales/',
  models: '/src/models/',
  pages: '/src/pages/',
  theme: '/src/theme/',
  services: '/src/services/',
  utils: '/src/utils/'
};

if (isDevelopment) {
  console.log('\n\n==== ALIAS =====\n');
  console.log(alias);
  console.log('\n==== /ALIAS =====\n\n');
}

module.exports = { alias };
