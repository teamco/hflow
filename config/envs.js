import { readFileSync } from 'fs';

const {
  env: {
    DEBUG = true,
    NODE_ENV = 'development',
    CORPORATE_PROXY = false,
    HOME_ENV = false,
    VERSION = readFileSync('./VERSION', 'utf8')
  },
  versions: {
    node: NODE_VERSION
  },
  arch: ARCH,
  platform: OS_PLATFORM
} = process;

const isDevelopment = NODE_ENV === 'development';

const envs = {
  CORPORATE_PROXY,
  DEBUG,
  HOME_ENV,
  NODE_ENV,
  NODE_VERSION,
  ARCH,
  OS_PLATFORM,
  VERSION
}

if (isDevelopment) {
  console.log('\n\n==== ENV =====\n');
  console.log(envs);
  console.log('\n==== /ENV =====\n\n');
}

export default envs