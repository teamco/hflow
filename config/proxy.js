const colors = require('colors');

const { RUNTIME_CONFIG } = require('../src/services/config/runtime.config');
const { API } = require('../src/services/config/api.config');
const { cloudinaryAPI } = require('../src/services/config/cloudinary.config');

const { NODE_ENV, CORPORATE_PROXY } = process.env;

let { SERVER_URL, SERVER_PORT, API_NS } = RUNTIME_CONFIG();

if (CORPORATE_PROXY) {
  SERVER_URL = null;
}

/**
 * @constant
 * @param {boolean} [debug]
 * @return {{onProxyRes(*, *, *): void, onError(*, *, *): void, logLevel: string, pathRewrite(*, *): void,
 *     onProxyReq(*, *, *): void}}
 */
const debugProps = (debug = true) => {
  return debug ? {
    logLevel: 'debug',
    pathRewrite(path, req) {
      console.info(path, req.url);
    },
    onError(err, req, res) {
      console.error(err);
      res.status(500);
      res.json({ error: 'Error when connecting to remote server.' });
    },
    onProxyReq(proxyReq, req, res) {
      console.info('onProxyReq', proxyReq.host);
    },
    onProxyRes(proxyRes, req, res) {
      console.log('onProxyRes', proxyRes.host);
    }
  } : {};
};

const isDevelopment = NODE_ENV === 'development';

const proxyPops = {
  changeOrigin: true,
  secure: false,
  ws: false,
  target: `${SERVER_URL}:${SERVER_PORT}/`,
  ...debugProps(isDevelopment)
};

const _proxy = CORPORATE_PROXY ? {} : {
  [`${API_NS}/${API.auth.token}`]: { ...proxyPops },
  [`${API_NS}/${API.auth.refresh}`]: { ...proxyPops },
  [`${API_NS}/${API.users.store}`]: { ...proxyPops },
  [`${API_NS}/${API.features.store}`]: { ...proxyPops },
  [`${API_NS}/${API.campaigns.store}`]: { ...proxyPops },
  [`${API_NS}/${API.schedulers.store}`]: { ...proxyPops },
  [`${API_NS}/${API.apartments.store}`]: { ...proxyPops },
  [`${API_NS}/${API.addresses.store}`]: { ...proxyPops },
  [`${API_NS}/${API.views.store}`]: { ...proxyPops },
  [`${API_NS}/${API.likes.store}`]: { ...proxyPops },
  [`${API_NS}/${API.subscriptions.store}`]: { ...proxyPops },
  [`${API_NS}/${cloudinaryAPI.signature}`]: { ...proxyPops },
};

/**
 * @export
 * @constant
 * @return {{[p: string]: {onProxyRes(*, *, *): void, onError(*, *, *): void, logLevel: string, changeOrigin: boolean,
 *     secure: boolean, ws: boolean, pathRewrite(*, *): void, onProxyReq(*, *, *): void, target: string}|{onProxyRes(*,
 *     *, *): void, onError(*, *, *): void, logLevel: string, changeOrigin: boolean, secure: boolean, ws: boolean,
 *     pathRewrite(*, *): void, onProxyReq(*, *, *): void, target: string}}}
 */
const getProxy = () => {
  console.log('\n\n==== PROXY =====\n');
  console.log(colors.green('NODE_ENV:'), NODE_ENV);
  console.log(colors.green('SERVER_URL:'), `${SERVER_URL}:${SERVER_PORT}`);
  console.log(colors.cyan('Under corporate proxy?'), `[${CORPORATE_PROXY ? 'Yes' : 'No'}]\n`);
  isDevelopment && console.log(colors.green('PROXY:'), _proxy);
  console.log('\n==== /PROXY =====\n\n');

  return _proxy;
};

module.exports = { proxy: getProxy() };