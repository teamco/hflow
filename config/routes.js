const colors = require('colors');

const { MAIN_ROUTES } = require('./routes/main.routes');

const { NODE_ENV } = process.env;

const isDevelopment = NODE_ENV === 'development';
const mainRoutes = MAIN_ROUTES();

const print = (route) => console.log(colors.green(route.path), '=>', colors.cyan(route.component));

function routesList() {
  console.log('\n\n==== ROUTES =====\n');
  const admins = [];
  const profiles = [];

  mainRoutes.routes.forEach(r => {
    console.log(colors.green(r.path), '=>', colors.cyan(r.component));

    if (r?.routes?.length) {
      r.routes.forEach(route => {
        if (route.path.match(new RegExp('/admin'))) {
          admins.push(route);
        } else if (route.path.match(new RegExp('/profile'))) {
          profiles.push(route)
        } else {
          print(route);
        }
      });
    }
  });

  console.log('\n==== PROFILE\n');
  profiles.forEach(r => print(r));

  console.log('\n==== ADMIN\n');
  admins.forEach(r => print(r));

  console.log('\n==== /ROUTES =====\n\n');
}

isDevelopment && routesList();

module.exports = { routes: [mainRoutes] };
