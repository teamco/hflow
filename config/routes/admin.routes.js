const { ERRORS } = require('./error.routes');

const wrappers = ['@/wrappers/auth.admin'];

/**
 * @export
 * @param {string} [adminPath]
 * @return {{path: string, routes: [{path: string, component: string, breadcrumb: string, wrappers: string[], exact: boolean},{path: string, component: string, breadcrumb: string, wrappers: string[], exact: boolean},{path: string, component: string, breadcrumb: string, wrappers: string[], exact: boolean},{path: string, component: string, breadcrumb: string, wrappers: string[], exact: boolean},{path: string, component: string, breadcrumb: string, wrappers: string[], exact: boolean},null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null], component: string, breadcrumb: string, exact: boolean}}
 * @constructor
 */
const ADMIN_ROUTES = (adminPath = '/admin') => {
  const adminErrors = ERRORS(adminPath, '');

  return {
    exact: false,
    path: adminPath,
    component: '@/layouts/app',
    breadcrumb: 'route.admin',
    routes: [
      {
        exact: true,
        path: adminPath,
        breadcrumb: 'route.home',
        component: '@/pages/home',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/users`,
        component: '@/pages/users',
        breadcrumb: 'route.users',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/profile`,
        component: '@/pages/users/[user]/profile',
        breadcrumb: 'route.userProfile',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/users/:user`,
        component: '@/pages/users/[user]/profile',
        breadcrumb: 'route.userProfile',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/users/:user/notifications`,
        component: '@/pages/notifications',
        breadcrumb: 'route.notifications',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/businesses`,
        component: '@/pages/users/[user]/businesses',
        breadcrumb: 'route.businesses',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/users/:user/businesses`,
        component: '@/pages/users/[user]/businesses',
        breadcrumb: 'route.businesses',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/businesses/:business`,
        component: '@/pages/users/[user]/businesses/[business]',
        breadcrumb: 'route.businessEdit',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/businesses/:business`,
        component: '@/pages/users/[user]/businesses/[business]/show',
        breadcrumb: 'route.businessShow',
        wrappers
      },
      // {
      //   exact: true,
      //   path: `${adminPath}/users/:user/businesses/:business/edit`,
      //   component: '@/pages/users/[user]/businesses/[business]',
      //   breadcrumb: 'route.businessEdit',
      //   wrappers
      // },
      {
        exact: true,
        path: `${adminPath}/users/:user/businesses/:business`,
        component: '@/pages/users/[user]/businesses/[business]',
        breadcrumb: 'route.businessShow',
        wrappers
      },
      // {
      //   exact: true,
      //   path: `${adminPath}/users/:user/businesses/:business/`,
      //   component: '@/pages/users/[user]/businesses/[business]/show',
      //   breadcrumb: 'route.businessShow',
      //   wrappers
      // },
      {
        exact: true,
        path: `${adminPath}/users/:user/businesses/:business/users`,
        component: '@/pages/users/[user]/businesses/[business]/users',
        breadcrumb: 'route.businessUsers',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/roles/user`,
        component: '@/pages/roles/userRoles',
        breadcrumb: 'route.userRoles',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/roles/business`,
        component: '@/pages/roles/businessRoles',
        breadcrumb: 'route.businessRoles',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/roles/component`,
        component: '@/pages/roles/componentRoles',
        breadcrumb: 'route.componentRoles',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/roles/ability`,
        component: '@/pages/roles/abilityRoles',
        breadcrumb: 'route.abilityRoles',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/roles/manager`,
        component: '@/pages/roles/rolesManager',
        breadcrumb: 'route.rolesManager',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/businessTypes`,
        component: '@/pages/business/types',
        breadcrumb: 'route.businessTypes',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/campaigns`,
        component: '@/pages/campaigns',
        breadcrumb: 'route.campaigns',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/campaigns/:campaign`,
        component: '@/pages/campaigns/[campaign]',
        breadcrumb: 'route.campaign',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/properties`,
        component: '@/pages/properties',
        breadcrumb: 'route.properties',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/properties/:property`,
        component: '@/pages/properties/[property]',
        breadcrumb: 'route.property',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/subscriptions`,
        component: '@/pages/subscriptions',
        breadcrumb: 'route.subscriptions',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/subscriptions/:subscription`,
        component: '@/pages/subscriptions/[subscription]',
        breadcrumb: 'route.subscription',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/schedulers`,
        component: '@/pages/schedulers',
        breadcrumb: 'route.schedulers',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/subscriptionTypes`,
        component: '@/pages/subscriptions/types',
        breadcrumb: 'route.subscriptionTypes',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/currencies`,
        component: '@/pages/currencies',
        breadcrumb: 'route.currencies',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/durationTypes`,
        component: '@/pages/durationTypes',
        breadcrumb: 'route.durationTypes',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/featureTypes`,
        component: '@/pages/subscriptions/features/types',
        breadcrumb: 'route.featureTypes',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/features`,
        component: '@/pages/subscriptions/features',
        breadcrumb: 'route.features',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/features/:feature`,
        component: '@/pages/subscriptions/features/[feature]',
        breadcrumb: 'route.feature',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/notifications`,
        component: '@/pages/notifications',
        breadcrumb: 'route.notifications',
        wrappers
      },
      {
        exact: true,
        path: `${adminPath}/logs`,
        component: '@/pages/userLogs',
        breadcrumb: 'route.userLogs',
        wrappers
      },
      ...adminErrors
    ]
  };
};

module.exports = { ADMIN_ROUTES };
