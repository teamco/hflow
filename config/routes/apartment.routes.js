/**
 * @export
 * @return [{}]
 * @constructor
 */
const APARTMENT_ROUTES = (mainPath, wrappers) => {

  return [
    {
      exact: true,
      path: `${mainPath}/profile/apartments`,
      breadcrumb: 'route.profile.apartments',
      component: '@/pages/landing/profile/apartments',
      wrappers
    },
    {
      exact: true,
      path: `${mainPath}/profile/:profile/apartments`,
      breadcrumb: 'route.profile.apartments',
      component: '@/pages/landing/profile/apartments',
      wrappers
    },
    {
      exact: true,
      path: `${mainPath}/profile/apartments/:apartment`,
      component: '@/pages/landing/profile/apartments/[apartment]',
      breadcrumb: 'route.profile.apartments',
      wrappers
    },
    {
      exact: true,
      path: `${mainPath}/profile/:profile/apartments/:apartment`,
      component: '@/pages/landing/profile/apartments/[apartment]',
      breadcrumb: 'route.profile.apartments',
      wrappers
    }
  ];
};

module.exports = { APARTMENT_ROUTES };
