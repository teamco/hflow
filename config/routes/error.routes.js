/**
 * @export
 * @param {string} [path]
 * @param {string} [page]
 * @param {string} [errorPath]
 * @returns {[]}
 */
const ERRORS = (path = '', page = '/landing/errors', errorPath = '/errors') => [
  {
    component: `@/pages${page}/warning`,
    breadcrumb: 'route.pageWarning',
    path: `${path}${errorPath}/warning`
  },
  {
    component: `@/pages${page}/500`,
    breadcrumb: 'route.page500',
    path: `${path}${errorPath}/500`
  },
  {
    component: `@/pages${page}/403`,
    breadcrumb: 'route.page403',
    path: `${path}${errorPath}/403`
  },
  {
    component: `@/pages${page}/404`,
    breadcrumb: 'route.page404',
    path: `${path}${errorPath}/404`
  },
  {
    component: `@/pages${page}/404`,
    breadcrumb: 'route.page404',
    path: `*`
  }
];

module.exports = { ERRORS };
