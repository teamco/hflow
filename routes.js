export const routes = [
  {
    exact: false,
    path: '/',
    component: '@/layouts/landing',
    routes: [
      {
        exact: true,
        path: '/',
        breadcrumb: 'route:home',
        component: '@/pages/landing'
      },
      {
        exact: true,
        path: '/profile',
        breadcrumb: 'route:profile',
        component: '@/pages/landing/profile'
      },
      {
        exact: false,
        path: '/admin',
        component: '@/layouts/app',
        breadcrumb: 'route:admin',
        routes: [
          {
            exact: true,
            path: '/admin/finishSignUp/:user',
            breadcrumb: 'route:finishSignUp',
            component: '@/pages/finishSignUp'
          },
          {
            exact: true,
            path: '/admin',
            breadcrumb: 'route:home',
            component: '@/pages/home',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/users',
            component: '@/pages/users',
            breadcrumb: 'route:users',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/profile',
            component: '@/pages/users/[user]/profile',
            breadcrumb: 'route:userProfile',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/users/:user',
            component: '@/pages/users/[user]/profile',
            breadcrumb: 'route:userProfile',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/users/:user/notifications',
            component: '@/pages/notifications',
            breadcrumb: 'route:notifications',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/businesses',
            component: '@/pages/users/[user]/businesses',
            breadcrumb: 'route:businesses',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/users/:user/businesses',
            component: '@/pages/users/[user]/businesses',
            breadcrumb: 'route:businesses',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/businesses/:business',
            component: '@/pages/users/[user]/businesses/[business]',
            breadcrumb: 'route:businessEdit',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/businesses/:business',
            component: '@/pages/users/[user]/businesses/[business]/show',
            breadcrumb: 'route:businessShow',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          // {
          //   exact: true,
          //   path: '/admin/users/:user/businesses/:business/edit',
          //   component: '@/pages/users/[user]/businesses/[business]',
          //   breadcrumb: 'route:businessEdit',
          //   wrappers: [
          //     '@/wrappers/auth.wrapper'
          //   ]
          // },
          {
            exact: true,
            path: '/admin/users/:user/businesses/:business',
            component: '@/pages/users/[user]/businesses/[business]',
            breadcrumb: 'route:businessShow',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          // {
          //   exact: true,
          //   path: '/admin/users/:user/businesses/:business/',
          //   component: '@/pages/users/[user]/businesses/[business]/show',
          //   breadcrumb: 'route:businessShow',
          //   wrappers: [
          //     '@/wrappers/auth.wrapper'
          //   ]
          // },
          {
            exact: true,
            path: '/admin/users/:user/businesses/:business/users',
            component: '@/pages/users/[user]/businesses/[business]/users',
            breadcrumb: 'route:businessUsers',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/manageRoles',
            component: '@/pages/userRoles',
            breadcrumb: 'route:manageRoles',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/businessTypes',
            component: '@/pages/business/types',
            breadcrumb: 'route:businessTypes',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/campaigns',
            component: '@/pages/campaigns',
            breadcrumb: 'route:campaigns',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/campaigns/:campaign',
            component: '@/pages/campaigns/[campaign]',
            breadcrumb: 'route:campaign',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/subscriptions',
            component: '@/pages/subscriptions',
            breadcrumb: 'route:subscriptions',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/subscriptions/:subscription',
            component: '@/pages/subscriptions/[subscription]',
            breadcrumb: 'route:subscription',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/subscriptionTypes',
            component: '@/pages/subscriptions/types',
            breadcrumb: 'route:subscriptionTypes',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/currencies',
            component: '@/pages/currencies',
            breadcrumb: 'route:currencies',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/durationTypes',
            component: '@/pages/durationTypes',
            breadcrumb: 'route:durationTypes',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/featureTypes',
            component: '@/pages/subscriptions/features/types',
            breadcrumb: 'route:featureTypes',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/features',
            component: '@/pages/subscriptions/features',
            breadcrumb: 'route:features',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/features/:feature',
            component: '@/pages/subscriptions/features/[feature]',
            breadcrumb: 'route:subscriptionPref',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/notifications',
            component: '@/pages/notifications',
            breadcrumb: 'route:notifications',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            exact: true,
            path: '/admin/logs',
            component: '@/pages/userLogs',
            breadcrumb: 'route:userLogs',
            wrappers: [
              '@/wrappers/auth.wrapper'
            ]
          },
          {
            component: '@/pages/warning',
            breadcrumb: 'route:pageWarning',
            path: '/admin/errors/warning'
          },
          {
            component: '@/pages/500',
            breadcrumb: 'route:page500',
            path: '/admin/errors/500'
          },
          {
            component: '@/pages/403',
            breadcrumb: 'route:page403',
            path: '/admin/errors/403'
          },
          {
            component: '@/pages/404',
            breadcrumb: 'route:page404',
            path: '/admin/errors/404'
          }
        ]
      }
    ]
  }
];
