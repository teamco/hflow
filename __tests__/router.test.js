import { routes } from '../config/routes';

it('Correct routes are exported', () => {
  expect(routes.map(({ name, path }) => (
      `${name}: ${path}`
  ))).toMatchSnapshot();
});
