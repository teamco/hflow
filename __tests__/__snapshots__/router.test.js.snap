// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correct routes are exported 1`] = `
[
  "undefined: /",
]
`;
